main.py

```diff
@@ -149,9 +149,8 @@ async def serve_console_lines(self, lines: Union[int, None]) -> str:
                 yield json.dumps({"line": line}) + "\n"
         else:
             # Copy the scrollback buffer so we don't modify it
-            copy = self.process.scrollback_buffer.copy()
-            relevant = copy[-lines:]
-            for line in relevant:
+            copy = self.process.scrollback_buffer[-lines:].copy()
+            for line in copy:
```

Comment:
This copy could be removed as it was only there when I had to reverse a deque which would mutate

Reply:
in this case the `copy` variable could be removed entirely:
```suggestion
            for line in self.process.scrollback_buffer[-lines:]:
```